source/PluginDev/Assets/GooglePlayGames/Editor/GPGSAndroidSetupUI.cs

```diff
@@ -86,14 +100,26 @@ public void DoSetup()
         /// </summary>
         /// <param name="appId">App identifier.</param>
         /// <param name="nearbySvcId">Optional nearby connection serviceId</param>
-        public static bool PerformSetup(string appId, string nearbySvcId)
-        {
-            // check for valid app id
-            if (!GPGSUtil.LooksLikeValidAppId(appId))
```

Comment:
Can this be optional?  So if the game does not need the token, this does not need to be configured? (And it will be backwards compatible).